angular/PdfViewer/Text-Markup-Annotation.md

```diff
@@ -50,21 +50,21 @@ export class DefaultComponent {
 
 The colorpicker control is provided in the text markup annotation tools to select the desired color for the text markup annotation to be added in the document.
 
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img2.png)
+![Adding notes in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img2.png)
```

Comment:
Adding text markup annotations in Angular PDF Viewer

Reply:
Updated